src/models/Settings.js

```diff
@@ -314,6 +314,9 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
     },
 
     setAuthClient: function (authClient) {
+      if (authClient && authClient.options && authClient.options.headers) {
```

Comment:
Can we move this part into its own function? `setLanguageForWidget` maybe? So that anyone who's using the widget can just comment that out if needed.